client/lib/analytics/index.js

```diff
@@ -19,6 +19,8 @@ var config = require( 'config' ),
 	_superProps,
 	_user;
 
+import { retarget } from 'lib/analytics/ad-tracking';
```

Comment:
Tests are unhappy with this import (If you export a noop from the ad-tracking file tests pass), which likely means that we need to mock more items in the ad-tracking file.